src/lib/provable/test/bitwise.unit-test.ts

```diff
@@ -183,6 +193,18 @@ await equivalentAsync({ from: [maybeField, maybeField], to: field }, { runs })(
   }
 );
 
+await equivalentAsync({ from: [maybeField, maybeField], to: field }, { runs })(
+  (x, y) => {
+    if (x >= 2n ** 64n || y >= 2n ** 64n)
+      throw Error('Does not fit into 64 bits');
+    return x & y;
```

Comment:
why do you return `&` in this case but `Bitwise.or` in the next?

Reply:
Good catch! That should be `|` not `&`. Interestingly, it doesn't fail the unit test. Any ideas, @mitschabaude?

Reply:
my only idea is that you use such a small number of `runs` that with high likelyhood, one of the two inputs is bigger than $2^64$ in every single run, so you get an error every time.

try it with `uint(64)` as the random generator instead of `maybeField`!

Reply:
I think you can actually remove this unit test that creates a proof, and stick with the ones above that only run a circuit and check constraints

The proof one will make CI slower and provides hardly any additional value

Reply:
While developing smart contracts, sometimes it works without proofs but fails with them. Isn't it beneficial to test with proofs. Is it different for primitives?

Reply:
It's different here yeah, for various reasons.

For one, we are already testing many extremely similar methods with proofs in this file, the additional surface covered is funny